package/govuk/objects/_template.scss

```diff
@@ -11,8 +11,8 @@
     // Prevent automatic text sizing, as we already cater for small devices and
     // would like the browser to stay on 100% text zoom by default.
     -webkit-text-size-adjust: 100%;
-       -moz-text-size-adjust: 100%;
-            text-size-adjust: 100%;
+    -moz-text-size-adjust: 100%;
+    text-size-adjust: 100%;
```

Comment:
Feels like Prettier is fighting Autoprefixer here? Does it make senes to run Prettier on our `package`, or should we exclude it?

Reply:
Ah nice spot, think this is just a hangover from before it was ignored
https://github.com/alphagov/govuk-frontend/pull/3201/files#diff-b640b344ee7f3f03d2a443795a5d0708ef50e2e6e34214109ab2aad13ad6ba98

I'll remove it!